### PR TITLE
feat(sms.options.blacklist): move blacklisted senders to options

### PR DESCRIPTION
**Pull request by Steffy29**
Ref: UXCT-189

### CODE REVIEW

File: packages/manager/modules/sms/src/options/blacklist/telecom-sms-options-blacklist.html [hunk 1]
```diff
@@ -29,24 +29,24 @@ <h1 data-translate="sms_senders_blacklisted_manage_title"></h1>
             <button
                 type="button"
                 class="btn btn-default"
-                data-ng-disabled="SmsSendersBlacklistedCtrl.getSelection().length === 0 || SmsSendersBlacklistedCtrl.blacklists.isDeleting"
-                data-ng-click="SmsSendersBlacklistedCtrl.deleteSelectedBlacklist()"
+                data-ng-disabled="SmsOptionsBlacklistCtrl.getSelection().length === 0 || SmsOptionsBlacklistCtrl.blacklists.isDeleting"
```
Comment (FredericEspiau) [inline packages/manager/modules/sms/src/options/blacklist/telecom-sms-options-blacklist.html]: Could you name it `$ctrl` instead ?

File: packages/manager/modules/sms/src/options/blacklist/index.js [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+import controller from './telecom-sms-options-blacklist.controller';
+import template from './telecom-sms-options-blacklist.html';
+
+const moduleName = 'ovhManagerSmsOptionsBlacklist';
+
+angular.module(moduleName, []).config(($stateProvider) => {
+  $stateProvider.state('sms.service.options.blacklist', {
+    url: '/blacklist',
+    views: {
+      'smsView@sms.service': {
```
Comment (FredericEspiau) [inline packages/manager/modules/sms/src/options/blacklist/index.js]: Use a component instead please :)

Comment (Steffy29) [reply packages/manager/modules/sms/src/options/blacklist/index.js]: next time ;-)

Comment (marie-j) [reply packages/manager/modules/sms/src/options/blacklist/index.js]: As you are improving this section,I agree with @FredericEspiau that a nice improvement would be to turn this into a component.

Comment (Steffy29) [reply packages/manager/modules/sms/src/options/blacklist/index.js]: Done, with component and routing

**Comment by FredericEspiau** [comment]
Awesome job :)

**Pull request merged**